templates/bet/index.ts

```
@@ -0,0 +1,65 @@
+import type { BaseConfig, BaseStorage, BaseTemplate } from '@/lib/types'
+import { ChainKey } from '@/sdk/viem'
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)